compiler/control/OMROptions.cpp

```diff
@@ -1200,6 +1200,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"traceSamplingJProfiling",          "L\ttrace samplingjProfiling",                     TR::Options::traceOptimization, samplingJProfiling, 0, "P"},
    {"traceSEL",                         "L\ttrace sign extension load",                    TR::Options::traceOptimization, signExtendLoads, 0, "P"},
    {"traceSequenceSimplification",      "L\ttrace arithmetic sequence simplification",     TR::Options::traceOptimization, expressionsSimplification, 0, "P"},
+   {"traceSequentialStoreSimplification", "L\ttrace sequential load or store simplification", TR::Options::traceOptimization, sequentialStoreSimplification, 0, "P"},
```

Comment:
Can this be placed under a `J9_PROJECT_SPECIFIC` guard to avoid having to move the definition of `sequentialStoreSimplification` into OMR? I think doing that should avoid the need for a coordinated merge between OMR and OpenJ9.

Reply:
I can make this changes.